Decred Wallet/Features/App Launch/StartScreenViewController.swift

```diff
@@ -47,7 +47,7 @@ class StartScreenViewController: UIViewController {
         }
         
         if AppDelegate.walletLoader.isWalletCreated {
-            self.label.text = "Opening wallet..."
+            self.label.text = "openingWallet".localized
```

Comment:
A quick thought, can we place these `Localizable.strings` references in an enum? E.g.:

```
enum LocalizedStrings: String {
     openingWallet = NSLocalizedString("openingWallet", comment: "")
}

....

self.label.text = LocalizedStrings.openingWallet
```

Reply:
I agree. It would be more readable and easy to work with.

Reply:
not sure that will work ,the raw value for enum must be literal and not referenced .